spec/interactors/create_response_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+RSpec.describe CreateResponse do
+  let(:rack_request) { ActionController::TestRequest.new('RAW_POST_DATA' =>  '{"message":"Hello World"}') }
+  let(:bucket)       { Bucket.create(name: 'My Bucket') }
+  let(:request) do
+    result = CreateRequest.call(bucket: bucket, rack_request: rack_request) # stub it?
+    result.request
+  end
+
+  describe '#call' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.